CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Next release
 
+#Next
```

Comment:
Not something to rework perse, but adding the `#nex` heading was not required. `#Next release` is on line 1